src/containers/toputilizers/TopUtilizersReducer.js

```diff
@@ -8,16 +8,19 @@ export const INITIAL_STATE:Immutable.Map<*, *> = Immutable.fromJS({
   entitySetId: '',
   entitySet: {},
   associations: [],
-  entityTypes: [],
   topUtilizersDetailsList: {},
   topUtilizersResults: [],
-  isGettingResults: false
+  isGettingResults: false,
+  associationDetails: Immutable.Map()
 });
 
 export default function reducer(state :Immutable.Map<*, *> = INITIAL_STATE, action :Object) {
   switch (action.type) {
-    case actionTypes.GET_ENTITY_TYPES_REQUEST:
-      return state;
+    case actionTypes.GET_ENTITY_SET_REQUEST:
+      return state.set('entitySetId', action.entitySetId);
```

Comment:
I was doing that since entitySetId is a part of the state and I was trying to minimize changes to how this is all set up, but actually I'm not sure that's even being used at all/is needed, so I can just get rid of it